presto-docs/src/main/sphinx/connector/cassandra.rst

```diff
@@ -192,3 +192,85 @@ This table can be described in Presto::
+-----------
+
+Data types
+^^^^^^^^^^
+
+The data types mappings are as follows:
+
+================  ======
+Cassandra         Presto
+================  ======
+ASCII             VARCHAR
+BIGINT            BIGINT
+BLOB              VARBINARY
+BOOLEAN           BOOLEAN
+DECIMAL           DOUBLE
+DOUBLE            DOUBLE
+FLOAT             DOUBLE
+INET              VARCHAR(45)
+INT               INTEGER
+LIST<?>           VARCHAR
+MAP<?, ?>         VARCHAR
+SET<?>            VARCHAR
+TEXT              VARCHAR
+TIMESTAMP         TIMESTAMP
+TIMEUUID          VARCHAR
+VARCHAR           VARCHAR
+VARIANT           VARCHAR
+================  ======
+
+Any collection (LIST/MAP/SET) can be designated as FROZEN, and the value is
+mapped to VARCHAR. Additionally, blobs have the limitation that they cannot be empty.
+
+Types not mentioned in the table above are not supported (e.g. tuple or UDT).
+
+Metadata maintenance
+^^^^^^^^^^^^^^^^^^^^
+Metadata is retrieved asynchronously when querying, and then refreshed
+asynchronously in intervals defined by the property
+``cassandra.schema-refresh-interval``. Schema refreshes occur using the
+number of threads defined in the property ``cassandra.max-schema-refresh-threads``.
+Metadata entries expire after the time specified in ``cassandra.schema-cache-ttl``.
+
+In a typical use case, ``cassandra.schema-refresh-interval`` should be not
+greater than ``cassandra.schema-cache-ttl``, unless the latter is equal to ``0s``.
+
+For example, for ``schema-refresh-interval=10s`` and ``schema-cache-ttl=30s``,
+the refresh will look like:
+
+================  ======================================================================================
+Time after query  Action
+================  ======================================================================================
+0s                start building metadata index asynchronously
+1-10s             index is not modified, except when a background thread triggered earlier finishes work
+10s               start refreshing metadata index asynchronously
+11-20s            index is not modified, except when a background thread triggered earlier finishes work
+20s               start refreshing metadata index asynchronously
+21-30s            index is not modified, except when a background thread triggered earlier finishes work
+30s               discard cache
+================  ======================================================================================
+
+.. note::
+
+    An outdated cache may result in wrong results. For example, a primary key
+    added after the latest refresh will not be visible in the results.
+    If your use case entails frequent addition of data to Cassandra, consider lowering the
+    value of ``cassandra.schema-cache-ttl`` (sometimes even to ``0s``). Be aware that this
+    may cause a significant performance degradation.
+
+
+Miscellaneous
+^^^^^^^^^^^^^
+
+* ``cassandra.schema-refresh-interval`` cannot be set to the value ``0s``.
```

Comment:
push-down for clustering keys is also not supported. also, I don't think that secondary indexes are supported?